internal/cloud/cloud.go

```diff
@@ -101,6 +97,28 @@ func Shell(configDir string) error {
 	return shell(username, vmHostname, configDir)
 }
 
+func enactGithubUsernameStep() string {
+	stepGithubUsername := stepper.Start("Detecting your Github username...")
```

Comment:
What do you think of making the username detection silent? When it fails to detect my username it looks like something went wrong.